src/porepy/viz/exporter.py

```diff
@@ -180,6 +182,183 @@ def __init__(
         # Parameter to be used in several occasions for adding time stamps.
         self._padding = 6
 
+    def import_from_vtu(
+        self, keys: Union[str, list[str]], file_names: Union[str, list[str]], **kwargs
+    ) -> None:
+        """
+        Import state variables from vtu file.
```

Comment:
Perhaps comment that it is assumed the file is generated by PorePy, follows pp naming conventions etc.? I'm not sure exactly what I mean, perhaps I want to say that kick-starting a simulation from outside PorePy using this function is for the brave.

Reply:
Initially I was thinking of adding a map (dict) allowing to address names previously used in some vtu file, but different than intended to use in the current simulation. However, to better focus on the mere importer functionality, I decided to omit any distractions. For practical use, some discussion could be anyhow helpful, as I also see possibilities to interact with initial times etc. So many different components may come together here. But first we would have to decide what we would want to use this for.